src/main/java/plugily/projects/buildbattle/boot/PlaceholderInitializer.java

```diff
@@ -90,7 +87,8 @@ private String getTheme(Player player, IPluginArena arena) {
         if(!(pluginArena instanceof GuessArena)) {
           return theme;
         }
-        if(theme.equalsIgnoreCase(new MessageBuilder("SCOREBOARD_THEME_UNKNOWN").asKey().build())) {
+          assert theme != null;
```

Comment:
any reason for assertion here?
it most likely won't be used during debugging/development stages anyway